libs/obs/include/mrpt/obs/CObservation3DRangeScan_project3D_impl.h

```diff
@@ -11,6 +11,9 @@
 
 #include <mrpt/core/round.h>  // round()
 
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
```

Comment:
Ey! Red alarm here! :-)

We can't assume PCL is installed, since PCL is an **optional** dependency for MRPT.
I think you could remove those headers and all should build nice. Why did you add them? 
For the templates for build, it's enough to have forward declarations, and in any case, the PCL headers included **in the user code** (the caller that includes this file). 

So, please, remove those two includes.